client/src/Components/HomePage/HomeNewsScreen.tsx

```diff
@@ -0,0 +1,105 @@
+import { RootStackParamList } from '../../types/navigation'
+import NewsForm from './NewsForm/NewsForm';
+import { observer } from 'mobx-react-lite'
+import Navigation from '../Navigation/Navigation'
+import axios from 'axios'
+import { NewsItem } from '../../types/NewsItem'
+import getNewsStore from '../../stores/newsStore'
+import CustomText from '../../CustomComponents/CustomText'
+import CustomView from '../../CustomComponents/CustomView'
+import getThemeStore from '../../stores/themeStore';
+import { darkMode, lightMode } from '../colors/colors';
+import { Dimensions } from 'react-native';
+
+const HomeNewsScreen : React.FC = observer(() => {
+    const navigation = useNavigation<NavigationProp<RootStackParamList>>();
+
+    const newsInformations = async() : Promise<void> => {
+        const IP_ADDRESS : string = "192.168.100.126"
+        try
+        {
+          const response = await axios.post<NewsItem[]>(`http://${IP_ADDRESS}:3000/api/news`)
+          console.log("Home News Screen :   ",response.data)
+          getNewsStore().setNewsList(response.data);          
+        } catch ( error ) {
+          console.error("Failed Fetching News Information" , error);
+        }
+      }
+      useEffect(() =>{
+        newsInformations();
+        console.log("NewsList in the store:", getNewsStore().newsList.get())
+      },[])
+
+      const theme = getThemeStore().isDarkThemeEnabled.get() ? darkMode : lightMode;
+      const windowWidth = Dimensions.get("window").width;
+      const windowHeight = Dimensions.get("window").height;
+  return (
+   <ScrollView style={{
+        backgroundColor: theme.backGroundColor,
+        width: windowWidth,
+        height: windowHeight
+    }}>
+        <Navigation />
+            <CustomView
+                style={{
+                    display: 'flex',
+                    justifyContent: 'flex-end',
+                    alignItems: 'flex-end',
+                    marginTop: 10,
+                    marginRight: 10
+                }}
+            >
+                <CustomView
+                        style={{
+                            display: 'flex',
+                            justifyContent:'center',
+                            alignSelf:'center',
+                            alignItems: 'center'
+                        }}
+                        
+                    >
+                        <CustomText
+                            style={{
+                                color: theme.fontColor
+                            }}
+                            fontSize={25}
+                            fontWeight='500'
+                        >
+                            The number of News is {getNewsStore().newsCount.get()}
+                        </CustomText>
+                    </CustomView>
+                <TouchableOpacity onPress={()=>navigation.navigate("AddNewsPopUp")}>
+                    
+                    <CustomView
```

Comment:
split this render components into multiple parts

Reply:
DONE ✅